CMakeLists.txt

```diff
@@ -620,13 +632,36 @@ if (USE_NCNN)
   include_directories(${NCNN_INC_DIR})
 endif()
 
+if (USE_TENSORRT)
+  find_package(Protobuf REQUIRED)
```

Comment:
It seems we are already using Protobuf for Caffe, TF and Pytorch, sometimes by forcing it up as it is included into TF and Pytorch typically. My question would be at this stage, do we want to move the `find_package` somewhere outside the conditional and/or do we need it actually ?

Reply:
- here I use system protobuf because in most tensorRT cases, it would be the only backend used (an so the only available protobuf). 
- as i use FindProtobuf.cmake, i also use cmake helper protobuf_generate_cpp. So getting rid of it does not work (or needs rewrite of src/CMakeLists)
- compiling caffe + tensorrt works with current cmakelists 

so I am not sure of what we shoud do here :)

Reply:
finally changed a a lot CMakeLists in order not to use find_package and use the same hacks as for caffe2/TF already included protobufs

Reply:
done,  tested ok